content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,31 @@
+---
+
+This page outlines guidelines for migrating from [couch2pg](https://github.com/medic/cht-couch2pg) to the data pipeline based on CHT Sync and CHT Pipeline. One of the main changes in this flow is separating the syncing process from the data transformation, with CHT Pipeline and DBT now handling the latter. This migration requires DBT models with CHT Pipeline instead of SQL views and tables. One thing to note is that the schema for CHT Sync differs from couch2pg, so DBT models will not directly replace the SQL views and tables. For instructions on how to get started with DBT models, refer to the [DBT Models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})).
+
+## Key Considerations
+- **Server resources**: To minimize downtime, running both couch2pg and CHT Sync in parallel during the migration process is recommended. With this in mind, ensure that the server and database resources are sufficient to handle the load.
+- **DBT Modelling**: Avoid the temptation to model new DBT models after existing SQL views and tables. Instead, take the opportunity to re-evaluate the data needs and design new models that are more efficient and effective. Think of what data needs to be shown and how it should be shown in data visualization tools and use that to guide the design of the new models.
+<!-- TODO: Link to docs for testing DBT models once they are ready -->
+- **Testing**: After migrating, thoroughly test the new DBT models to ensure that they are working as expected.
+- **Feedback**: Provide any feedback and create issues for errors or bugs encountered in the [CHT Sync](https://github.com/medic/cht-sync) and [CHT Pipeline](https://github.com/medic/cht-pipeline/) repositories to improve the tools.
+
+## Migration Steps
+1. **Plan the migration**: Determine the scope of the migration, including the data sources, the data models, and the data transformations. Identify the existing SQL views, tables, and dashboards and assess what data you want to visualize. 
+1. **Set up CHT Sync**: Follow the instructions in the [Local CHT Sync Setup](({{< relref "apps/guides/data/analytics/setup" >}})) guide to set up CHT Sync locally.
+1. **Build DBT models**:  Use the dedicated guidelines to build [DBT models](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})) for the data you want to visualize.
+1. **Deploy CHT Sync**: Once the DBT models are tested locally and working as expected, deploy CHT Sync in production. Follow the instructions in the [Production CHT Sync Setup](({{< relref "apps/guides/data/analytics/production" >}})) guide to set up CHT Sync in production. It is recommended that CHT Sync be run in parallel with couch2pg during the migration process. This minimises disruption to users of the existing dashboards because they can continue to use the existing data while the new pipeline is being set up. It also makes it easier when testing the new pipeline to compare the data from couch2pg.
```

Comment:
```suggestion
1. **Deploy CHT Sync**: Once the DBT models are tested locally and working as expected, deploy CHT Sync in production. Follow [the instructions to set up CHT Sync in production](({{< relref "apps/guides/data/analytics/production" >}})). It is recommended that CHT Sync be run in parallel with couch2pg during the migration process. This minimises disruption to users of the existing dashboards because they can continue to use the existing data while the new pipeline is being set up. It also makes it easier to compare the data from couch2pg when testing the new pipeline.
```